napari/utils/misc.py

```diff
@@ -257,7 +258,7 @@ def formatdoc(obj):
         obj.__doc__ = obj.__doc__.format(
             **{**frame.f_globals, **frame.f_locals}
         )
-        return obj
+        return obj  # noqa: TRY300
```

Comment:
Is there a reason why this cannot work like in other places with the `else` block?

Reply:
Because else works only if there is `except` clause. 

But it could be simply moved after `finally`

Opened issue: https://github.com/charliermarsh/ruff/issues/3632

Reply:
Wow, that was fast, already merged :P